src/api/app/assets/stylesheets/webui/application/icons.scss

```diff
@@ -120,6 +120,7 @@
 .icons-page_white_add { width: 16px; height: 16px; background: image-url('icons_sprite.png') -342px -400px no-repeat; }
 .icons-page_white_delete { width: 16px; height: 16px; background: image-url('icons_sprite.png') -362px -400px no-repeat; }
 .icons-page_white_get { width: 16px; height: 16px; background: image-url('icons_sprite.png') -382px -400px no-repeat; }
+.icons-page_white_put { width: 16px; height: 16px; background: image-url('icons_sprite.png') -382px -400px no-repeat; }
```

Comment:
Note the px coordinates (eg. ` -382px -400px `) at the end of each line. Each of them points to a single icon on the [sprite file](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/assets/images/icons_sprite.png).